client/js/lounge.js

```diff
@@ -170,6 +170,10 @@ $(function() {
 		$("#sign-in").remove();
 
 		var id = data.active;
+
+		// Set it right away before triggering a click so that "open" event is not emitted
```

Comment:
Marking the page as read removes the unread marker, right? If so, I think this is fine and the page should be left as unread.